tests/e2e/e2e_setup_test.go

```diff
@@ -233,7 +231,8 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		}
 	}
 
-	if !s.skipUpgrade {
+	switch {
```

Comment:
Were you trying things here? I'm not sure what this change is about

Reply:
i think its a cherry pick issue, my bad